src/mono/netcore/System.Private.CoreLib/src/System/Math.Mono.cs

```diff
@@ -64,6 +64,8 @@ public partial class Math
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern double Sin(double a);
 
+        public static (double Sin, double Cos) SinCos(double x) => (Sin(x), Cos(x));
```

Comment:
@marek-safar, could I get sign off from you on the Mono changes here and could you let me know if there should be a tracking issue for any hook ups required on the Mono side?

Reply:
@jkotas, do you know if there is an alias to tag for new APIs that Mono may need to react to in order to ensure everything is getting correctly tracked/handled?

Reply:
I do not think there is alias for that. You have all the right people in the list of reviewers already. Since nobody raise any objections, I think it is fine to merge.